accounts/templates/emails/resend_confirmation.html

```diff
@@ -2,11 +2,11 @@
   Dear {{ researcher_name }},
 </p>
 <p>
-    Here is a link to verify your email address on the OSF: {{ osf_url}}/resend/.
+    Here is a link to verify your email address on the OSF: {{ osf_url}}resend/.
```

Comment:
I'm sure 99.999% of email clients will turn this and the one below it into a link, but shouldn't it be inside of an anchor tag?